app/src/main/java/com/henagle/www/kanban/MainActivity.java

```diff
@@ -175,50 +147,9 @@ public void onClick(DialogInterface dialog, int id) {
         addCardDialog = builder.create();
     }
 
-    /*
-    Custom adapter for card list
-     */
-    private class CardListAdapter extends BaseAdapter {
```

Comment:
I moved this adapter into the `DeckFragment` class (and renamed it `DeckAdapter`, since that is now responsible for handling the ListView of `Card` objects. The flow now looks like this:

`MainActivity` has a `ViewPager`
`ViewPager` has a `ViewPagerAdapter` that knows which `DeckFragment` to load based on where the user is
`DeckFragment` has a `ListView` that it populates Cards with using a `DeckAdapter`